src/main/java/com/example/ajaxonboardingproject/config/MemoryBeanPostProccesor.kt

```diff
@@ -28,20 +30,21 @@ class MemoryCheckBeanPostProcessor : BeanPostProcessor {
             val enhancer = Enhancer()
             enhancer.setSuperclass(originalBeanClass)
             enhancer.setCallback(MethodInterceptor { _, method, args, _ ->
-                val before = computeUsedMemory()
-                val result = method.invoke(bean, *args)
-                val after = computeUsedMemory()
-                if ((after - before) > MAX_MEMORY_USAGE) {
-                    throw ResponseStatusException(HttpStatus.BAD_REQUEST, "Memory usage is too high")
+                val payloadSize = ByteArrayOutputStream().use { byteArrayOutputStream ->
+                    ObjectOutputStream(byteArrayOutputStream).use { objectOutputStream ->
+                        args.forEach { objectOutputStream.writeObject(it) }
+                        objectOutputStream.flush()
+                        byteArrayOutputStream.size()
+                    }
```

Comment:
It would be nice to extract well-named function:
```kotlin
private fun countSize(args...) : Long { // Maybe you will give a better name
   //...
}
```

Reply:
Great idea!